_data/releases.yml

```diff
@@ -21,6 +21,38 @@
 
 # 2.7 series
 
+- version: 2.7.0-rc1
+  date: 2019-12-18
+  post: /en/news/2019/12/18/ruby-2-7-0-rc1-released/
+  total_files: 4163
```

Comment:
possibly more clearer:

```yaml
stats:
  files_changed: 4163
  insertions: 226280
  deletions: 99449
```

(I could adjust that at a later time.)

Reply:
I fix it in next release

Reply:
Already fixed for this post.